app/src/main/kotlin/net/corda/cli/application/logger/LoggerStream.kt

```diff
@@ -0,0 +1,68 @@
+package net.corda.cli.application.logger
+
+import java.io.OutputStream
+import java.io.PrintStream
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.io.ByteArrayOutputStream
+
+enum class LogLevel {
+    TRACE, DEBUG, INFO, WARN, ERROR
```

Comment:
```suggestion
    INFO, ERROR
```
Given that we only ever create `LoggerStream` with `INFO` and `ERROR`...